apps/web/ui/layout/sidebar/user-dropdown-old.tsx

```
@@ -0,0 +1,143 @@
+            href="/account/settings"
+            onClick={() => setOpenPopover(false)}
+          />
+          {...!isPartnerPage
+            ? [
+                <UserOption
+                  as={Link}
+                  label="Refer and earn"
+                  icon={Gift}
+                  href="/account/settings/referrals"
+                  onClick={() => setOpenPopover(false)}
+                />,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)